📖  Add details about lifecycle/active in the documentation (#3347)
kubernetes-sigs/cluster-api: CONTRIBUTING.md

If you're new to the project and want to help, but don't know where to start, we have a semi-curated list of issues that
should not need deep knowledge of the system. [Have a look and see if anything sounds
interesting](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

> Before starting to work on the issue, make sure that it is not already taken up by someone else. The issues already taken up by someone have a [lifecycle/active](https://github.com/kubernetes-sigs/cluster-api/labels/lifecycle%2Factive) label. 

Reviewer: ```suggestion
Before starting to work on the issue, make sure that it doesn't have a [lifecycle/active](https://github.com/kubernetes-sigs/cluster-api/labels/lifecycle%2Factive) label. If the issue has been assigned, reach out to the assignee. 
```

Author: Done

Alternatively, read some of the docs on other controllers and try to write your own, file and fix any/all issues that
come up, including gaps in documentation!

## Contributing a Patch

1. If you haven't already done so, sign a Contributor License Agreement (see details above).

> 1. If working on an issue, signal the other contributors that you are starting to work on it by leaving a comment. 

Reviewer: ```suggestion
1. If working on an issue, signal other contributors that you are actively to work on it using `/lifecycle active`. 
```

Author: Done

interesting](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
Alternatively, read some of the docs on other controllers and try to write your own, file and fix any/all issues that
come up, including gaps in documentation!

> Also, before starting to work on the issue, make sure that it is not already taken up by someone else. The issues already taken up by someone have a [lifecycle/active](https://github.com/kubernetes-sigs/cluster-api/labels/lifecycle%2Factive) label. 

Reviewer 2: What about moving this sentence before Alternatively? I think that it is more appropriate there...

Reviewer 2: Also, what about adding the following sentence before line 47 so we are describing this practice also when someone starts to contribute to a patch
 
1. If working on an issue,  signal to the other contributes that you starting to work on it by applying the

Author: Sure.

Author: Can anyone apply this label by comenting `\lifecycle active` ?

Author: I made some changes :D
